server/controllers/resetController.js

```diff
@@ -0,0 +1,75 @@
+import { config } from 'dotenv';
+import authHelper from '../helpers/authHelpers';
+
+config();
+
+/**
+ * This class contains all methods
+ * required to handle all
+ * password reset and update.
+ */
+class ResetController {
+  /**
+     * This method find the user email request.
```

Comment:
Consider changing the comment to '**This method checks if the user is already registered prior to resetting the password**'.

Reply:
updated